modules/cudev/include/opencv2/cudev/grid/detail/transform.hpp

```diff
@@ -179,6 +179,22 @@ namespace grid_transform_detail
         dst(y, x) = saturate_cast<DstType>(op(src1(y, x), src2(y, x)));
     }
 
+    // transformSimple, 2 outputs
+
+    template <class SrcPtr1, class SrcPtr2, typename DstType1, typename DstType2, class BinOp1, class BinOp2, class MaskPtr>
+    __global__ void transformSimple(const SrcPtr1 src1, const SrcPtr2 src2, GlobPtr<DstType1> dst1, GlobPtr<DstType2> dst2,
```

Comment:
Can you add a comment regarding the purpose of such overloads?

I suppose, you want to skip propagation of tuple/array down the call chain, but I might be wrong.

Reply:
The branch is needed for `gridTransformBinary` in polar_cart.cu. The functions requires 2 inputs and 2 outputs at a time. The previous version called `gridTransformTuple` with known cuda namespace clash.